scout/utils/ensembl_biomart_clients.py

```diff
@@ -1,35 +1,33 @@
 import logging
-from typing import Callable, Dict, Iterator
+from typing import Dict, Iterator
 
 import requests
-from schug.load.biomart import EnsemblBiomartClient
-from schug.load.ensembl import fetch_ensembl_exons, fetch_ensembl_genes, fetch_ensembl_transcripts
-from schug.models.common import Build as SchugBuild
 
 LOG = logging.getLogger(__name__)
+SCHUG_BASE = "https://schug-stage.scilifelab.se"
```

Comment:
Stage because the is [a bug](https://github.com/Clinical-Genomics/schug/issues/64) in prod, which I am fixing right now